index.js

```diff
@@ -14,17 +16,26 @@ function fastatic(options) {
 	loader.setSpinnerString(18);
 	loader.start();
 
-	const logger = new Logger(config.logLevel);
+	const result = copy(config.src, config.temp)
+		.then(() => parseAll(config))
+		.then(() => stats(config))
+		.then(output => console.log(output))
+		.then(() => loader.stop())
+		.then(() => Promise.all([
```

Comment:
I agree this would be the setup if we do more than `compareFileSize`, but without it, I find this overcomplicating the module. So leave out `spread` and instead use:
```javascript
.then(() => ({
  fileSize: compareFileSize(config.src, config.temp)
});
```